.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+<!--
+**Thanks for contributing to Fury.**
+
+**If this is your first time opening a PR on fury, you can refer to [CONTRIBUTING.md](https://github.com/apache/incubator-fury/blob/main/CONTRIBUTING.md).**
+
+Contribution Checklist
+
+    - The **Apache Fury (incubating)** community has restrictions on the naming of pr titles. You can also find instructions in [CONTRIBUTING.md](https://github.com/apache/incubator-fury/blob/main/CONTRIBUTING.md).
+
+    - Fury does not accept PRs that degrade performance. If the PR you submit will have an impact on performance, please Benchmark it yourself first.
```

Comment:
```suggestion
    - Fury has a strong focus on performance. If the PR you submit will have an impact on performance, please benchmark it first and provide the benchmark result here.
```
I personly feel `Fury does not accept PRs that degrade performance` is a little aggressive.

Reply:
Updated.